### PR TITLE
Hotfix/video

### PR DESCRIPTION
**Pull request by deven98**
Added video fixes

### CODE REVIEW

File: packages/stream_chat_flutter/lib/src/message_widget.dart [hunk 1]
```diff
@@ -142,6 +142,8 @@ class MessageWidget extends StatefulWidget {
   /// Function called when quotedMessage is tapped
   final OnQuotedMessageTap onQuotedMessageTap;
 
+  final Map<String, VideoPackage> videoPackages;
```
Comment (imtoori) [inline packages/stream_chat_flutter/lib/src/message_widget.dart:145]: Let's add a comment here

File: packages/stream_chat_flutter/lib/src/message_list_view.dart [hunk 2]
```diff
@@ -239,6 +239,8 @@ class _MessageListViewState extends State<MessageListView> {
 
   final MessageListController _messageListController = MessageListController();
 
+  Map<String, VideoPackage> videoPackages = {};
```
Comment (imtoori) [inline packages/stream_chat_flutter/lib/src/message_list_view.dart:242]: This can be final

**Review by imtoori — CHANGES_REQUESTED** [review]

File: packages/stream_chat_flutter/lib/src/message_list_view.dart (hunk 2 above)
Comment (deven98) [reply packages/stream_chat_flutter/lib/src/message_list_view.dart:242]: It's in the state, I don't think this matters that much.

File: packages/stream_chat_flutter/lib/src/message_widget.dart (hunk 1 above)
Comment (deven98) [reply packages/stream_chat_flutter/lib/src/message_widget.dart:145]: Yep, adding it.

**Review by imtoori — APPROVED** [review]
LGTM

**Pull request merged**